tests/test_dse.py

```diff
@@ -2874,7 +2874,7 @@ def test_opcounts(self, space_order, expected):
 
     @switchconfig(profiling='advanced')
     @pytest.mark.parametrize('space_order,exp_ops,exp_arrays', [
-        (4, 122, 6), (8, 221, 7)
+        (4, 122, 6), (8, 235, 7)
```

Comment:
why the increase? variable density on by default? that feels weird

Reply:
No it's because of the space order I think. THe model used to be setup with space order `4` while setting up the second case with space order `8` so not sure if it was creating a few low order derivatives.
All the other opcount case stay the same the FD hasn't changed (b is never setup in oss for tti)  it's just the space order change